sfcasts/ep2-fundamentals/services-yaml.md

```diff
@@ -1,105 +1,46 @@
-watch I'll approve this. This actually still works just fine. Let me undo that. There
-we go. The only reason we have a service.Yaml file is just organization. It feels
-good to have one file to configure your services. The truly important thing is that
-all this config lives under the services key. In fact, near the top of this file,
-notice there's an empty parameters key. And you might remember from earlier in
-cache.yaml We created a parameter's key there to register a parameter. We could have,
-it's really up to us to decide where, where we want to define this parameter. We can
-do it in 'cache.yaml', or if we want to, we can copy this And move it over to
-'services.yaml'
-
-And `cache.yaml`, I'll also grab the, `when@dev` delete that and paste that into
-`services.yaml` That on a technical level makes no difference. And our app still
-works actually like this better services and parameters are a global idea in your
-app. So it's nice to organize them all in one file. All right, next, the only reason
-we've written any code at the bottom of our services.yaml file is to hit to the
-container. What to pass to the non autowire is debug argument, but there's a more
-automatic way to solve these problematic arguments.
+When Symfony first boots up, it needs to get the full list of all of the services that should be in the container. That includes the service ID, its class name, and all of its constructor arguments. The first and biggest source of services are *bundles*. If you run
 
+```terminal
+php bin/console debug:container
+```
+
+the vast majority of these services come from bundles. The *second* place the container gets services from is our *code*. To learn about our services, Symfony reads `services.yaml`. And when Symfony starts parsing the first line of this file, nothing in our `/src` directory has been registered as a service in the container. This is really important. Adding our classes to the container is, in fact, this file's *job*, and the way it does that is pretty amazing. Let's walk through it!
+
+Notice that the service config is under a `services` key. And the first key under here is `_defaults`. This is a pretty fancy config key that defines some default options that will be added to *all* services that are registered in this file. So *every* service will *automatically* have `autowire: true` and `autoconfigure: true`.
+
+Earlier, we set this up to manually register a service. We'll talk more about that in a minute. Right now, I want to focus on the `bind` option we saw. Services can, in fact, have a *bunch* of different options, including `autowire` and `autoconfigure`. So it would be *totally* legal for me to say, `autowire: true` and `autoconfigure: true` right here. This would work just fine, but thanks to the `_defaults` section, those are *not* needed. the `_defaults` says:
+
+`Unless it's been specifically overridden on a
+service, set 'autowire' and 'autoconfigure' to
+'true' for all services in this file.`
```

Comment:
For these "quote" sections (which you've been identifying perfectly), put them into block quotes without the "ticks" (well, you can tick specific technical words, but not everything):

```
> Unless it's been overridden on a specific service, set `autowire` and
> `autoconfigure` to `true` for all services in this file.
```